src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -430,6 +433,17 @@ message InstanceConfig {
   }
 }
 
+message AuthConfig {
```

Comment:
yeah... let's get rid of it and just hard code a path.
java is verbose enough as it is. the less code/complexity we add the better.

Reply:
No. Hard-coding paths makes my life harder than it needs to be. From an operational standpoint, I need the ability to change the path because I haven't determined where the secret will be mounted yet and that path may need to change.